### ISSUE TITLE
No block performance metrics produced

### ISSUE THREAD

**Issue by ajsutton** [Consensys-Incorporated/teku#4581]
### Description
Teku is no longer reporting block performance metrics either on the console or via metrics. Likely caused by #4543

**Comment by ajsutton** [comment]
Happening on our pyrmont nodes but I'm not able to reproduce with a local interop mode node.

**Comment by ajsutton** [comment]
Also happening on our prater nodes.

**Comment by ajsutton** [comment]
Ah, it's reporting on the current epoch instead of the just completed one.  For a node running every validator on a minimal network it gives:
```
Epoch Event *** Epoch: 9, Justified checkpoint: 7, Finalized checkpoint: 6, Finalized root: cf1398..1efb
Attestation performance: epoch 7, expected 64, produced 64, included 64 (100%), distance 1 / 1.00 / 1, correct target 64 (100%), correct head 64 (100%)
Block performance: epoch 9, expected 1, produced 1, included 0 (0%)
```

but it should be expecting 8 blocks with 8 produced and 8 included.
